### ISSUE TITLE
Failure/Error: DatabaseCleaner[:sequel].strategy = :truncation

### ISSUE THREAD

**Issue by ms-ati** [DatabaseCleaner/database_cleaner-sequel#27]
This error appeared in 2.0.1

```
Failure/Error: DatabaseCleaner[:sequel].strategy = :truncation
    ArgumentError: wrong number of arguments (given 0, expected 1)
```

```
     # /usr/packages/ruby-2.7.2/gems/database_cleaner-sequel-2.0.1/lib/database_cleaner/sequel/truncation.rb:6:in `initialize'
     # /usr/packages/ruby-2.7.2/gems/database_cleaner-core-2.0.1/lib/database_cleaner/cleaner.rb:85:in `new'
     # /usr/packages/ruby-2.7.2/gems/database_cleaner-core-2.0.1/lib/database_cleaner/cleaner.rb:85:in `create_strategy'
     # /usr/packages/ruby-2.7.2/gems/database_cleaner-core-2.0.1/lib/database_cleaner/cleaner.rb:45:in `strategy='
     # ./spec/rails_helper.rb:74:in `block (2 levels) in <top (required)>'
```

**Comment by ms-ati** [comment]
It appears that in 2.0.1 this was the breaking change: https://github.com/DatabaseCleaner/database_cleaner-sequel/commit/c70a12f54e7fa412d26ae783f9938bb358dd21de

**Comment by ms-ati** [comment]
It appears to be fixed in this commit: https://github.com/DatabaseCleaner/database_cleaner-sequel/commit/7ad713fb746005f624b4b791d2f3c100a2002cde

**Comment by ms-ati** [comment]
Are we planning a 2.0.2 release? Should 2.0.1 be marked as a regression so that folks stop upgrading to it?

**Comment by etagwerker** [comment]
Just released v2.0.2. Thanks! 👍
